_i18n/en.yml

```diff
@@ -501,6 +501,11 @@ faq:
   avuln: >
     The Monero community has created a series of videos called "Breaking Monero", where potential Monero vulnerabilities are explored and discussed. There are 14 videos, with each exploring a different subject. Check out <a href="https://www.monerooutreach.org/breaking-monero/">the videos on monerooutreach.org</a>.
   vulnspotify: Available on Spotify as podcast
+  qasicresistance: "What is ASIC resistance? Why is it important?"
+  aasicresistance: ASICs are basically special computers created to do only one job, contrary to normal computers, which are made for general purpose. This characteristic makes ASICs very efficient for @mining.
+  aasicresistance1: The problem is that these devices are very expensive, consume a lot of electricity and can be afforded by few. This leads to few entities owning a big amount of the hashrate of the network, which is a serious threat to the security of the network itself. For example, if big ASIC operators collude and manage to gain the majority of the hashrate of the network, they could arbitrarily reject transactions.
```

Comment:
I don't think "consume a lot of electricity" fits as a problem. The general issue revolves around access from my view. Cost is part of that, especially on the production side. Another is insider preference/self-mining incentive.

Reply:
Removed.